pkg/deploy/che_configmap.go

```diff
@@ -225,3 +235,14 @@ func NewCheConfigMap(cr *orgv1.CheCluster, cheEnv map[string]string) *corev1.Con
 		Data: cheEnv,
 	}
 }
+
+func extraImagesConfig(cr *orgv1.CheCluster) map[string]string {
+	extraImages := map[string]string{
+		"CHE_WORKSPACE_PLUGIN__BROKER_INIT_IMAGE":    patchDefaultImageName(cr, cheWorkspacePluginBrokerInitImage),
+		"CHE_WORKSPACE_PLUGIN__BROKER_UNIFIED_IMAGE": patchDefaultImageName(cr, cheWorkspacePluginBrokerUnifiedImage),
+		"CHE_SERVER_SECURE__EXPOSER_JWTPROXY_IMAGE":  patchDefaultImageName(cr, cheServerSecureExposerJwtProxyImage),
+		"CHE_WORKSPACE_SIDECAR_IMAGE__PULL__POLICY":  cheWorkspaceSidecarImagePullPolicy,
```

Comment:
> CheWorkspaceSidecarImagePullPolicy
> CheDockerAlwaysPullImage

it seems you don't do anything with these values here since you already added them in the main config map building code.
Either we given them a specific value for airgap, or we can just remove them completely and the default will be used by the Che server from the `che.properties`.